Loop/Public/SwiftUI/SwiftUISubscription.swift

```diff
@@ -0,0 +1,32 @@
+import SwiftUI
+import Combine
+import ReactiveSwift
+
+@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+internal final class SwiftUISubscription<State, Event>: ObservableObject {
+    let objectWillChange = ObservableObjectPublisher()
+    var latestValue: State!
```

Comment:
Can't we initialise it with current state?

Reply:
Can we also use `@Published` here? Or is there something wrong with it?

Reply:
IUO is intentional to support both `@LoopBinding` and `@EnvironmentLoop`. There is no initial state to being with for `@EnvironmentLoop`, because the actual loop object does not come until the first call of `mutating func update()`.